mlrun/model_monitoring/applications/histogram_data_drift.py

```diff
@@ -189,21 +189,24 @@ def _get_general_drift_result(
         )
 
         status = self._value_classifier.value_to_status(value)
+
+        model_endpoint_stats = {
+            EventFieldType.CURRENT_STATS: json.dumps(
+                monitoring_context.sample_df_stats
+            ),
+            EventFieldType.DRIFT_MEASURES: json.dumps(
+                metrics_per_feature.T.to_dict()
+                | {metric.name: metric.value for metric in metrics}
+            ),
+            EventFieldType.DRIFT_STATUS: status.value,
+        }
+
         return mm_results.ModelMonitoringApplicationResult(
             name=HistogramDataDriftApplicationConstants.GENERAL_RESULT_NAME,
             value=value,
             kind=ResultKindApp.data_drift,
             status=status,
-            extra_data={
-                EventFieldType.CURRENT_STATS: json.dumps(
-                    monitoring_context.sample_df_stats
-                ),
-                EventFieldType.DRIFT_MEASURES: json.dumps(
-                    metrics_per_feature.T.to_dict()
-                    | {metric.name: metric.value for metric in metrics}
-                ),
-                EventFieldType.DRIFT_STATUS: status.value,
-            },
+            extra_data=model_endpoint_stats,
```

Comment:
you right, I'll remove it from the hist data drift

Reply:
and as a result we don't need both `DRIFT_STATUS` & `DRIFT_MEASURES` (your next 2 comments) at the moment. It will be calculated and stored according [to @royischoss ticket. ](https://iguazio.atlassian.net/browse/ML-7461)